src/glfw/video-driver.cpp

```diff
@@ -514,7 +519,11 @@ void GLFWVideoDriver::loop(Card* initial) {
             main_loop.draw();
             glfwSwapBuffers(_window);
         } else {
+#ifdef _MSC_VER
             std::this_thread::sleep_for(std::chrono::microseconds(10));
```

Comment:
Apparently this is missing from mingw even though it's a C++11 standard function.